extensions/markdown-language-features/src/commands/insertResource.ts

```diff
@@ -76,10 +76,10 @@ async function insertLink(activeEditor: vscode.TextEditor, selectedFiles: vscode
 	await vscode.workspace.applyEdit(edit);
 }
 
-function createInsertLinkEdit(activeEditor: vscode.TextEditor, selectedFiles: vscode.Uri[], insertAsMedia: boolean) {
+function createInsertLinkEdit(activeEditor: vscode.TextEditor, selectedFiles: vscode.Uri[], insertAsMedia: boolean, title = '', placeholderValue = 0) {
 	const snippetEdits = coalesce(activeEditor.selections.map((selection, i): vscode.SnippetTextEdit | undefined => {
 		const selectionText = activeEditor.document.getText(selection);
-		const snippet = createUriListSnippet(activeEditor.document, selectedFiles, {
+		const snippet = createUriListSnippet(activeEditor.document, selectedFiles, title, placeholderValue, {
```

Comment:
What's the difference between these new arguments and the existing `placeholderText` / `placeholderStartIndex` values passed in below?

Reply:
I did not want to change to existing functionality of insertResource.ts, so I wanted to keep placeholderText and placeholderString as they are. I made an edit in shared.ts that will hopefully make more sense.